client/src/Components/Students.js

```diff
@@ -37,6 +41,8 @@ const Students = (props) => {
 					fetchedData={fetchedData}
```

Comment:
I try to understand if this lines are generated by GitHub to illustrate what is in the props? Probably it is just githubs version of the view illustrating what props student component takes

Reply:
I think yes